.github/workflows/kube-linter-sample.yml

```diff
@@ -11,9 +11,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Scan repo
-        id: kube-lint-repo
-        uses: stackrox/kube-linter-action@v1.0.0
+
+      - name: Scan valid yaml with kube-linter
+        uses: stackrox/kube-linter-action@v1.0.2
         with:
-          directory: yaml
+          directory: sample/valid-yaml
```

Comment:
Seems worthwhile to add a config file in that directory as well.
I suggest using this one (save as `.kube-linter-config.yml` or so):
```yaml
# customChecks defines custom checks.
customChecks:
- name: "required-annotation-team"
  template: "required-annotation"
  params:
    key: "team"
  remediation: "Add a team annotation to your object"
checks:
  # if doNotAutoAddDefaults is true, default checks are not automatically added.
  doNotAutoAddDefaults: false

  # include explicitly adds checks, by name. You can reference any of the built-in checks.
  # Note that customChecks defined above are included automatically.
  include: []
  # exclude explicitly excludes checks, by name. exclude has the highest priority: if a check is
  # in exclude, then it is not considered, even if it is in include as well.
  exclude: []

```

Reply:
Thanks, done!